bin/compile

```diff
@@ -48,36 +38,12 @@ if [[ $CLEAR_CACHE -eq 1 ]] ; then
   rm -fr $CACHE_DIR/
 fi
 
-############# Library files ###############################
-
-if [ ! -e $CACHE_DIR/ghc-libs ]; then
-  LIBS_URL="https://github.com/PostgREST/postgrest-heroku/releases/download/v0.4.4.0/ghc-libs.tar.xz"
-  if curl --output /dev/null --silent --head --fail $LIBS_URL; then
-    echo "-----> Downloading and caching libraries for PostgREST"
-    mkdir -p $CACHE_DIR
-    curl -# -L $LIBS_URL | tar xJ -C $CACHE_DIR
-    cp -R $CACHE_DIR/ghc-libs $BUILD_DIR
-  else
-    echo "-----> Could not find prebuilt libraries"
-    exit 1
-  fi
-else
-  echo "-----> Restoring cached libraries"
-  cp -R $CACHE_DIR/ghc-libs $BUILD_DIR
-fi
-
 ############# PostgREST binary ################################
 
 BIN="postgrest-${POSTGREST_VER}"
 
-if ver20 || ver30 || ver40to42; then
-  RELEASE_VER="${POSTGREST_VER}"
-else
-  RELEASE_VER="v${POSTGREST_VER}"
-fi
-
 if [ ! -e $CACHE_DIR/$BIN ]; then
-  BIN_URL="https://github.com/PostgREST/postgrest/releases/download/v${POSTGREST_VER}/postgrest-${RELEASE_VER}-ubuntu.tar.xz"
+  BIN_URL="https://github.com/PostgREST/postgrest/releases/download/v${POSTGREST_VER}/postgrest-${RELEASE_VER}-linux-x64-static.tar.xz"
```

Comment:
A conditional for >= 6.0.0 to change to the linux-x64-static bin would also do here, I think.